packages/validation/src/Call/index.ts

```diff
@@ -1,14 +1,18 @@
+import { DateTime } from 'luxon';
 import * as Yup from 'yup';
 
-import { isValidDate } from '../util';
```

Comment:
I wonder could you just keep the method isValidDate in utils and just change the implementation there. Looks like that's easier to maintain,

Reply:
Yes I was thinking about that and you are right. Thanks